link/src/main/java/com/stripe/android/link/LinkViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.stripe.android.link
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.flatMapConcat
+import kotlinx.coroutines.flow.receiveAsFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+// This will be used as a base for all Link screen viewmodels to create a consistent architecture
```

Comment:
these comments are helpful for review! I think these work better as github comments for review only, and shouldn't be checked in. If you want to check in comments like this, you should add proper javadoc.